index.html

```diff
@@ -4,6 +4,19 @@
 		<meta charset="utf-8">
 		<meta http-equiv="X-UA-Compatible" content="IE=edge">
 		<title>Simulateur de coût d'embauche</title>
+		<style>
+			body {
+				font-family: 'Helvetica Neue', Arial, Helvetica, sans-serif;
+				line-height: 2.5;
+			}
+
+			/* Customize the widget's style */
+			body .SGMAPembauche {
```

Comment:
Webpack permits importing css in .js files by injecting them in the html header. So the widget's styles ends up as a `<style>` tag in `<head>`, *after* the original <styles>.